tests/Resources/ti.ui.animation.test.js

```diff
@@ -0,0 +1,420 @@
+			});
+
+			it('equals Ti.UI.Animation', () => {
+				should(instance.apiName).eql('Ti.UI.Animation');
+			});
+		});
+
+		describe('.autoreverse', () => {
+			it('is a Boolean', () => {
+				should(instance).have.a.property('autoreverse').which.is.a.Boolean();
+			});
+
+			it('defaults to false', () => {
+				should(instance.autoreverse).be.false();
+			});
+		});
+
+		describe('.backgroundColor', () => {
+			it('is a String', () => {
+				instance.backgroundColor = 'black';
+				should(instance)
+					.have.a.property('backgroundColor').which.is.a.String();
+			});
+		});
+
+		describe('.bottom', () => {
+			it('is a Number', () => {
+				instance.bottom = 0;
+				should(instance).have.a.property('bottom').which.is.a.Number();
+			});
+		});
+
+		describe('.center', () => {
+			it('is a Point', () => {
+				instance.center = { x: 50, y: 50 };
+				should(instance).have.a.property('center').which.is.an.Object();
+			});
+		});
+
+		describe('.color', () => {
+			it('is a String', () => {
+				instance.color = 'green';
+				should(instance).have.a.property('color').which.is.a.String();
+			});
+		});
+
+		describe('.curve', () => {
+			it('is a Number', () => {
+				instance.curve = Titanium.UI.ANIMATION_CURVE_EASE_IN;
+				should(instance).have.a.property('curve').which.is.a.Number();
+			});
+
+			it('is one of Ti.UI.ANIMATION_CURVE_EASE_*', () => {
+				instance.curve = Titanium.UI.ANIMATION_CURVE_EASE_IN_OUT;
+				should([
+					Ti.UI.ANIMATION_CURVE_EASE_IN,
+					Ti.UI.ANIMATION_CURVE_EASE_IN_OUT,
+					Ti.UI.ANIMATION_CURVE_EASE_OUT,
+					Ti.UI.ANIMATION_CURVE_LINEAR,
+				]).containEql(instance.curve);
+			});
+		});
+
+		describe.ios('.dampingRatio', () => {
+			it('is a Number', () => {
+				instance.dampingRatio = 0.8;
+				should(instance).have.a.property('dampingRatio').which.is.a.Number();
+			});
+		});
+
+		describe('.delay', () => {
+			it('is a Number', () => {
+				instance.delay = 200;
+				should(instance).have.a.property('delay').which.is.a.Number();
+			});
+		});
+
+		describe('.duration', () => {
+			it('is a Number', () => {
+				instance.duration = 200;
+				should(instance).have.a.property('duration').which.is.a.Number();
+			});
+		});
+
+		describe.android('.elevation', () => {
+			it('is a Number', () => {
+				instance.elevation = 3;
+				should(instance).have.a.property('elevation').which.is.a.Number();
+			});
+		});
+
+		describe('.height', () => {
+			it('is a Number', () => {
+				instance.height = 100;
+				should(instance).have.a.property('height').which.is.a.Number();
+			});
+		});
+
+		describe('.left', () => {
+			it('is a Number', () => {
+				instance.left = 0;
+				should(instance).have.a.property('left').which.is.a.Number();
+			});
+		});
+
+		describe('.opacity', () => {
+			it('is a Number', () => {
+				instance.opacity = 0.5;
+				should(instance).have.a.property('opacity').which.is.a.Number();
+			});
+		});
+
+		describe.ios('.opaque', () => {
```

Comment:
feels like this property should have a known/documented default? Or does it depend on the view it's applied to?